resume_2

```diff
@@ -1,186 +0,0 @@
-	  </ul>
-	    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-	 <script>
-	     $(".jobs").hide();
-	        $(".paid").on("click", function(){
-	            $(".jobs").show();
-	        });
-	        $(".paid").on("dblclick", function(){
-	            $(".jobs").hide();
-	        });
-	     $(".volt").hide();
-	        $(".unPaid").on("click", function(){
-	            $(".volt").show();
-	        });
-	        $(".unPaid").on("dblclick", function(){
-	            $(".volt").hide();
-	        });
-	     $(".refs").hide();
-	        $(".tellem").on("click", function(){
-	            $(".refs").show();
-	        });
-	        $(".tellem").on("dblclick", function(){
-	            $(".refs").hide();
-	        });
-	     $(".knows").hide();
-	        $(".learnt").on("click", function(){
-	            $(".knows").show();
-	        });
-	        $(".learnt").on("dblclick", function(){
-	            $(".knows").hide();
-	        });
-	     $(".cook").hide();
-	        $(".fud").on("click", function(){
-	            $(".cook").show();
-	        });
-	        $(".fud").on("dblclick", function(){
-	            $(".cook").hide();
-	        });
-	        $(".proder").hide();
-	        $(".airWaves").on("click", function(){
-	            $(".proder").show();
-	        });
-	        $(".airWaves").on("dblclick", function(){
-	            $(".proder").hide();
-	        });
-	        $(".teach").hide();
-	        $(".liluns").on("click", function(){
-	            $(".teach").show();
-	        });
-	        $(".liluns").on("dblclick", function(){
-	            $(".teach").hide();
-	        });
-	        $(".vegs").hide();
-	        $(".growem").on("click", function(){
-	            $(".vegs").show();
-	        });
-	        $(".growem").on("dblclick", function(){
-	            $(".vegs").hide();
-	        });
-	        $(".eros").hide();
-	        $(".artWork").on("click", function(){
-	            $(".eros").show();
-	        });
-	        $(".artWork").on("dblclick", function(){
-	            $(".eros").hide();
-	        });
-	        $(".coach").hide();
-	        $(".olymps").on("click", function(){
-	            $(".coach").show();
-	        });
-	        $(".olymps").on("dblclick", function(){
-	            $(".coach").hide();
-	        });
-	        $(".frankie").hide();
-	        $(".Sophia").on("click", function(){
-	            $(".frankie").show();
-	        });
-	        $(".Sophia").on("dblclick", function(){
-	            $(".frankie").hide();
-	        });
-	        $(".toms").hide();
-	        $(".Mike").on("click", function(){
-	            $(".toms").show();
-	        });
-	        $(".Mike").on("dblclick", function(){
-	            $(".toms").hide();
-	        });
-	        $(".chiefs").hide();
-	        $(".chefs").on("click", function(){
-	            $(".chiefs").show();
-	        });
-	        $(".chefs").on("dblclick", function(){
```

Comment:
repeating a lot of similar stuff so I thought there might be a different way of doing this. also it will make it a bit harder each time you want to add a new section or some different text because you'll need to add it in the HTML and the JS